README.md

```diff
@@ -1118,20 +1118,21 @@ In either case:
        an attribute when `self` is an ActiveRecord model: `self.guest = user`.
     3. Referencing the current instance's class: `self.class`.
 
-* <a name="freeze-constants"></a>When defining arrays and hashes meant to be
-    constants, make sure to call `freeze` on them. ([More on this][ruby-freeze].)
-    <sup>[[link](#freeze-constants)]</sup>
+* <a name="freeze-constants"></a>When defining strings, arrays or hashes meant to 
```

Comment:
Rubocop won't always be able to enforce things in the guide; the guide will often be more expansive than rubocop is capable of handling.